rust/lance-table/src/io/commit/dynamodb.rs

```diff
@@ -385,6 +388,8 @@ impl ExternalManifestStore for DynamoDBExternalManifestStore {
                     _ => None,
                 });
 
+                let e_tag = item.get("e_tag").and_then(|attr| attr.as_s().ok().cloned());
```

Comment:
Does dynamodb need any kind of migration for adding a new column?

Reply:
Only if you are creating new key columns / indices. For general columns, it's a schemaless document database.